tensorflow/core/framework/resource_mgr.cc

```diff
@@ -25,6 +25,26 @@ limitations under the License.
 
 namespace tensorflow {
 
+Status setOutputResourceHandle(OpKernelContext* context, int output_index,
```

Comment:
Can you name this SetOutputResourceHandle for more consistency with the surrounding code?